Faust Siu Proj 2

```diff
@@ -175,23 +175,33 @@ PROC LOESS data=rank_lenfol;
     ODS SELECT Fitplot;
 run;
 
-/*Sgplot los for age and sho*/
```

Comment:
New update for SGPLOT, this is with ranked log time plots. 

/*SGPLOT SCHOENFELD AGE los*/
proc sgplot data = rank_los; 
	loess x = loglos y = schage / clm; 
	title 'Age Residuals by Time';
run;

/*SGPLOT SCHOENFELD SHO los*/
proc sgplot data = rank_los; 
	loess x = loglos y = schsho / clm; 
	title 'SHO Residuals by Time';
run;

/*SGPLOT SCHOENFELD AGE lenfol*/
proc sgplot data = rank_lenfol; 
	loess x = loglenfol y = schage / clm; 
	title 'Age Residuals by Time';
run;


/*SGPLOT SCHOENFELD SHO lenfol*/
proc sgplot data = rank_lenfol; 
	loess x = loglenfol y = schsho/ clm; 
	title 'SHO Residuals by Time';
run;